src/api.js

```diff
@@ -4,14 +4,13 @@ const ncNewsApi = axios.create({
     baseURL: "https://be-nc-news-3me1.onrender.com/api/"
 });
 
-export const getArticles = (topic) => {
+export const getArticles = (topic, sort_by="created_at", order="DESC") => {
     if (topic) {
         return ncNewsApi.get(`articles?topic=${topic}`).then((response) => {
             return response.data;
         })
-        
     } else {
-        return ncNewsApi.get("articles").then((response) => {
+        return ncNewsApi.get(`articles?sort_by=${sort_by}&order=${order}`).then((response) => {
```

Comment:
This way is totally fine, but axios has its own way to deal with queries by passing a query object as the second argument, which will make it a bit easier. e.g.  `ncNewsApi.get( "articles" , { params: {topic, sort_by, order ...}})`